storey/table.py

```diff
@@ -1042,6 +1048,9 @@ def aggregate(self, time, value):
     def get_update_expression(self, old):
         return f'max({old}, {self.value})'
 
+    def aggregate_lua_script(self, vl1, vl2):
+        return f'type({vl1}) == "number" and math.max({vl1},{vl2}) or {vl2}'
```

Comment:
Not sure I understand this... Is this supposed to be the value or a condition? Some of the implementations have what seems to be a value, e.g. `f'{vl1}+{vl2}'`, and some have what seems to be a condition. Also the signature here doesn't match the one in the super class.

Reply:
neither a value or condition - it's part of lua script, to be joined with other parts of the script (see lua_script in _build_feature_store_lua_update_script)